lib/pbench/server/api/resources/query_apis/search_api.py

```diff
@@ -0,0 +1,201 @@
+from flask import jsonify
+from logging import Logger
+from typing import Any, AnyStr, Dict
+
+from pbench.server import PbenchServerConfig
+from pbench.server.api.resources.query_apis import (
+    ElasticBase,
+    Schema,
+    Parameter,
+    ParamType,
+    PostprocessError,
+)
+
+
+class SearchAPI(ElasticBase):
```

Comment:
If your endpoint is `index/search`, then `IndexSearch` would be a more consistent name.